lib/widgets/chat_balloon.dart

```diff
@@ -2,7 +2,7 @@ import 'package:flutter/material.dart';
 
 import 'package:have_you_heard/constants/colors.dart';
 
-Widget ChatBalloon(Text balloonHeader, Text balloonText) {
+Widget ChatBalloon(Text balloonHeader, RichText balloonText) {
```

Comment:
#14 added another use of ChatBalloon. https://github.com/yuumasato/have_you_heard/pull/14/files#diff-fa948c91c1957d09f74f0b975cbe66999cd4c0eba34c4c806ac931040d6470e8R61

I think you'll need to rebase and change the parameter type there too.

Reply:
Some other new users of ChatBalloon in master.